### PR TITLE
explain: Add promotion requirements

### PR DESCRIPTION
**Pull request by ihsavru**
Closes https://github.com/coala/corobo/issues/170

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: plugins/explain.py [hunk 1]
```diff
@@ -73,6 +73,15 @@ class Explain(BotPlugin):
                   'are sitting in front of the problem, not us. You will have '
                   'a much easier time solving it. That\'s why you should try '
                   'doing it first.'
```
Comment (meetmangukiya) [inline plugins/explain.py]: Missing a `comma` here

File: plugins/explain.py [hunk 2]
```diff
@@ -73,6 +73,15 @@ class Explain(BotPlugin):
...
+        'promotion': 'To become part of the coala developers team, there are a '
+                     'few steps you need to complete. The newcomer process is '
+                     'as follows:\nYou will start as a newcomer, which is kind '
+                     'of a trial. If you complete the following tasks, you will '
+                     'become a developer at coala:\n\nrun coala on a project '
```
Comment (meetmangukiya) [inline plugins/explain.py]: Please use `- ` so they are rendered as bullets

File: plugins/explain.py (hunk 1 above)
Comment (ihsavru) [inline plugins/explain.py]: I didn't quite get it. Where?

**Comment by meetmangukiya** [comment]
ack 652e252caa1a9f3d99c743e4e74cccab66df906c

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**